### PR TITLE
messaging: rabbitmq_binding, rabbitmq_exchange, rabbitmq_queue

### PR DESCRIPTION
**Pull request by manuel-sousa**
These modules allow for creation/deletion of exchanges, queues and bindings.

They use rabbitMQ REST API.

### CODE REVIEW

**Comment by bcoca** [comment]
could you please open a PR per module? makes it easier for us to revise and merge 1 by 1

File: messaging/rabbitmq_binding.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+module: rabbitmq_binding
+author: Manuel Sousa
+version_added: 1.5.4
```
Comment (resmo) [inline messaging/rabbitmq_binding.py:6]: should be '2.0'

File: messaging/rabbitmq_binding.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+                        "arguments": module.params['arguments']
+                    })
+                )
+        elif module.params['state'] == 'absent':
+            r = requests.delete( url, auth = (module.params['login_user'],module.params['login_password']))
+
+        if r.status_code == 204 or r.status_code == 201:
+            module.exit_json(
+                changed = True,
+                result = "Success",
```
Comment (resmo) [inline messaging/rabbitmq_binding.py:168]: exit_json is always success afaik.

**Comment by resmo** [comment]
good job. quite useful for me. Will test them.

**Comment by manuel-sousa** [comment]
I was about to do the 3 seperate pull rquests, is it still required?

**Comment by manuel-sousa** [comment]
As for the version, sorry i'm still using 1.5.4 that's why I used that as reference cause i knew it worked there;)

**Comment by resmo** [comment]
I do not work for ansible inc but @bcoca does. That's a yes. ☺

**Comment by manuel-sousa** [comment]
Ok, i'm making the change to the version and removing the success as commented and then submit the three of them separately. Thanks

**Comment by resmo** [comment]
@manuel-sousa @bcoca can be closed, modules merged in separete PRs.

**Pull request closed without merging**